dhall-json/tasty/Main.hs

```diff
@@ -55,7 +65,10 @@ testTree =
             , testDhallToJSON "./tasty/data/unionKeys"
             ]
         ]
-    where omissibleLists = JSONToDhall.defaultConversion{JSONToDhall.omissibleLists = True}
+    where
+        defaultConversion = JSONToDhall.defaultConversion
+        omissibleLists = defaultConversion{JSONToDhall.omissibleLists = True}
```

Comment:
This variable seems unused?!

Reply:
`omissibleLists` is used just before `Test.Tasty.testGroup "Errors"`

Reply:
Sorry, I had missed that it's a pre-existing definition.